public/script.js

```diff
@@ -5974,7 +6000,40 @@ export function cleanUpMessage(getMessage, isImpersonate, isContinue, displayInc
     return getMessage;
 }
 
-export async function saveReply(type, getMessage, fromStreaming, title, swipes, reasoning) {
+/**
+ * Adds an image to the message.
+ * @returns {Promise<void>}
+ */
+async function processImageAttachment(message, { parsedImage, imageUrl }) {
+    if (parsedImage?.image) {
+        return saveImageToMessage(parsedImage, message);
+    }
+
+    if (!imageUrl) {
+        return;
+    }
+
+    if (isDataURL(imageUrl)) {
+        const [mime, base64] = /^data:(.*?);base64,(.*)$/.exec(imageUrl).slice(1);
+        const url = await saveBase64AsFile(base64, message.name, Date.now().toString(), mime.split('/')[1]);
```

Comment:
Aren't we saving images in configurable char-folders now, not just based on the name?

And can we get some prefix for the image name? I know it's not like, that relevant, but just a timestamp is meh. `img_inline_123456`, or whatever.